README.md

```diff
@@ -167,5 +183,12 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+1.
+newLamp = Lamp.create(wishes_remaining:3) #id = 1
+newGenie = Genie.create(name:'Genie')
+2. newGenie.update(lamp_id:1)
+3. newLamp.update(wishes_remaining:1)
+4. jafarLamp = Lamp.create(wishes_remaining:3) #id = 2
+jafar = Genie.create(name:'Jafar',lamp_id:2)
+5. newLamp.update(wishes_remaining:nil)
```

Comment:
This isn't entirely correct: you virtually never want to hardcode IDs, or even use an `_id` field. Instead, you'd use `newGenie.update(lamp: newLamp)`. Let ActiveRecord take care of the IDs for you.